website/project/views/drafts.py

```diff
@@ -285,7 +294,7 @@ def get_metaschemas(*args, **kwargs):
     if include == 'latest':
         meta_schemas = RegistrationSchema.objects.get_latest_versions()
 
-    meta_schemas = sorted(meta_schemas, key=lambda x: METASCHEMA_ORDERING.index(x.name))
+    meta_schemas = sorted(meta_schemas, key=lambda x: sort_schemas(x))
```

Comment:
nit: no need for lambda since x is the argument, just `sorted(meta_schemas, key=sort_schemas)`